crates/duty-tracker/src/contract_manager.rs

```diff
@@ -1105,17 +1107,100 @@ impl ContractManagerCtx {
                 Ok(())
             }
 
-            OperatorDuty::PublishGraphNonces { deposit_txid } => {
-                let (pog, pog_conns) = PegOutGraph::generate(
+            OperatorDuty::PublishGraphNonces {
+                deposit_txid,
+                operator_p2p_key,
+            } => {
+                let Some(active_contract) = self.active_contracts.get(&deposit_txid) else {
+                    // this can only happen if some other operator is requesting nonces
+                    // and we have not yet observed the corresponding chain event.
+                    // so, it should be fine to ignore this error and allow the node to re-query our
+                    // node once we presumably observe the chain event.
+                    error!(
+                        ?deposit_txid,
+                        ?operator_p2p_key,
+                        "missing active contract for deposit txid"
+                    );
+                    return Ok(());
```

Comment:
Shouldn't we publish this again as gossip?
Otherwise the other operator will keep nagging you forever no?